superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -100,6 +100,20 @@ const MonospaceDiv = styled.div`
   white-space: pre-wrap;
 `;
 
+const ReturnedRows = styled.div`
+  font-size: 13px;
+  line-height: 24px;
+  .returnedRowsImage {
+    color: #ff7f43;
+    vertical-align: bottom;
+    margin-right: 8px;
+  }
+  .limitMessage {
+    color: #8e94b0;
+    margin-left: 8px;
```

Comment:
can we try to use the themeProvider here for the spacings and colors?

Reply:
Done! I couldn't find the right orange color though, should I add it in?

Reply:
Let's ask @steejay if the existing orange works: FF7F44

Reply:
@eschutho @AAfghahi existing works. nearly identical.